marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteDataTest.java

```diff
@@ -30,7 +34,9 @@
 
     @Test
     void ingestThreeFruitsWithBatchSizeOfTwo() throws IOException {
-        DataWriter<InternalRow> dataWriter = buildDataWriter("2", "/testFruit");
+        Map<String, String> params = getHubPropertiesAsMap();
```

Comment:
Pinging @SameeraPriyathamTadikonda  about this - for each workUnit input that the endpoint supports, we'll want to verify that it works correctly in the BulkIngestTest class, as that class has no knowledge of Spark.

The tests for our Spark connector should then focus on converting the params Map into the proper workUnit object. Those tests don't need to connect to ML - they can be very fast unit tests that live in a separate class that doesn't extend AbstractHubCoreTest. I'll put together a PR to demonstrate this and submit it against this branch. 

We'll still want one test in this project that writes data with every workUnit field populated, just to make sure that we're setting the correct workUnit fields.